dev/com.ibm.ws.security.wim.adapter.ldap_fat/build.gradle

```
@@ -138,6 +138,20 @@ autoFVT.doLast {
       into new File(autoFvtDir, 'publish/servers/' + server + '/dropins')
     }
   }
+  
+  /*
+   * Copy the truststore.p12 into test servers.
+   */
+  servers = [
+    'com.ibm.ws.security.wim.adapter.ldap.fat.tds.ssl',
+    'com.ibm.ws.security.wim.adapter.ldap.fat.ad.ssl'
+  ]
+  servers.each { server ->
+    copy { 
+      from project(':com.ibm.ws.security.wim.adapter.ldap_fat').file('resources/truststore.p12')
```

This should be from the unboundid project, not from this project. Delete the truststore.p12 from this project and change this to copy from the unboundID project.

```suggestion
      from project(':com.ibm.ws.com.unboundid').file('resources/truststore.p12')
```

---

Delete these files so that they are not in source control any longer:

>....ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.ad.ssl/truststore.p12
>....ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tds.ssl/truststore.p12

I also believe you copied the cert files into com.ibm.ws.security.wim.adapter.ldap.fat.ad.ssl, so move them to the unboundid project.